bundler-packages/resolutions/src/utils/resolveSafe.spec.ts

```
@@ -66,7 +66,7 @@ describe('resolveSafe', () => {
 		}
 		const error = await Effect.runPromise(flip(resolveSafe('./resolveSafe.spec.ts', './src/utils', safeFao(fao))))
 		expect(error).toMatchInlineSnapshot(
-			'[ExistsError: Unable to determine existence: Unable to determine existence: existsSync error]',
+			`[ResolveError: Failed to resolve]`,
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)